### PR TITLE
Add better output for the user to command 'console'.

### PR DESCRIPTION
**Pull request by kenden**
- when the instance is not found
 - when the stack is not found
 - when there are no instances in the region.

In those cases ``senza console`` just returned and displayed nothing, which was confusing for the user.
Add usage examples to ``console --help``.

Fix and add tests.

### CODE REVIEW

**Comment by hjacobs** [comment]
Thanks for your PR :smile:

**Comment by hjacobs** [comment]
:+1:

File: senza/cli.py [hunk 1]
```diff
@@ -1211,6 +1224,15 @@ def console(instance_or_stack_ref, limit, region, w, watch):
...
                         print_console(line)
+        if not found_match:
+            if stack_refs is None:
+                print("No EC2 instance with id '{}'.".format(instance_or_stack_ref[0]))
+            elif instance_or_stack_ref:
+                print("No stack matching '{}'{}.".format(
+                    stack_refs[0].name,
+                    " with version '{}'".format(stack_refs[0].version) if stack_refs[0].version is not None else ''))
+            else:
+                print("No EC2 instances in region '{}'.".format(region))
```
Comment (rafaelcaricio) [inline senza/cli.py]: Probably we should change the return code to 1 in all those cases...

Comment (kenden) [inline senza/cli.py]: @rafaelcaricio thanks. fix in https://github.com/zalando-stups/senza/pull/247

**Pull request merged**